teuthology/suite/run.py

```diff
@@ -32,7 +32,7 @@ class Run(object):
     __slots__ = (
         'args', 'name', 'base_config', 'suite_repo_path', 'base_yaml_paths',
         'base_args', 'package_versions', 'kernel_dict', 'config_input',
-        'timestamp', 'user',
+        'timestamp', 'user', 'flavor',
```

Comment:
Just for my understanding, why did you decide to make `flavor` a slot when `kernel_flavor` wasn't a slot?

Reply:
@kyr @zmc would it be otherwise in any case?